app/controllers/api/v1/organizations_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class OrganizationsController < Api::BaseController
+      def update
```

Comment:
Do we plan to add a route to show the current_organization?

Reply:
Product decided that we only need update ATM @vincent-pochet